### PR TITLE
Update addVirus.txt

### PR DESCRIPTION
**Pull request by trandnquang**

### CODE REVIEW

**Comment by PatrickTran2k5** [comment]
Ok wait for me checkout

File: addVirus.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 MUhahahahah
-I just add a virus
\ No newline at end of file
+I just add a virus
+blalala
```
Comment (PatrickTran2k5) [inline addVirus.txt:3]: Are you serious

Comment (trandnquang) [reply addVirus.txt:3]: O i a o

**Review by PatrickTran2k5 — APPROVED** [review]
Ok ok fine

**Pull request merged**